src/test/java/com/wikia/webdriver/common/enums/Mercury.java

```diff
@@ -0,0 +1,27 @@
+package com.wikia.webdriver.common.enums;
+
+/**
+ * @ownership Content X-Wing Wikia
+ */
+public class Mercury {
```

Comment:
What is the lacking part of this name ? Enum context is given by the parten package